src/components/hp.jsx

```diff
@@ -0,0 +1,293 @@
+import "./homepage.css";
+
+
+const features = [
+    {
+        name: 'Push to deploy',
+        description:
+            'Morbi viverra dui mi arcu sed. Tellus semper adipiscing suspendisse semper morbi. Odio urna massa nunc massa.',
+    },
+    {
+        name: 'SSL certificates',
+        description:
+            'Sit quis amet rutrum tellus ullamcorper ultricies libero dolor eget. Sem sodales gravida quam turpis enim lacus amet.',
+
+    },
+    {
+        name: 'Simple queues',
+        description:
+            'Quisque est vel vulputate cursus. Risus proin diam nunc commodo. Lobortis auctor congue commodo diam neque.',
+
+    },
+    {
+        name: 'Advanced security',
+        description:
+            'Arcu egestas dolor vel iaculis in ipsum mauris. Tincidunt mattis aliquet hac quis. Id hac maecenas ac donec pharetra eget.',
+
+    },
+]
+const featuress = [
+    {
+        name: 'Push to deploy',
+        description:
+            'Morbi viverra ',
+    },
+    {
+        name: 'SSL certificates',
+        description:
+            'Sit quis ',
+
+    },
+    {
+        name: 'Simple queues',
+        description:
+            'Quisque est ',
+
+    },
+    {
+        name: 'Advanced security',
+        description:
+            'Arcu egestas.',
+
+    },
+    {
+        name: 'Simple queues',
+        description:
+            'Quisque est ',
+
+    },
+    {
+        name: 'Advanced security',
+        description:
+            'Arcu egestas d',
+
+    },
+]
+const callouts = [
+    {
+        name: 'Desk and Office',
+        description: 'Work from home accessories',
+        imageSrc: 'https://tailwindui.com/img/ecommerce-images/home-page-02-edition-01.jpg',
+        imageAlt: 'Desk with leather desk pad, walnut desk organizer, wireless keyboard and mouse, and porcelain mug.',
+        href: '#',
+    },
+    {
+        name: 'Self-Improvement',
+        description: 'Journals and note-taking',
+        imageSrc: 'https://tailwindui.com/img/ecommerce-images/home-page-02-edition-02.jpg',
+        imageAlt: 'Wood table with porcelain mug, leather journal, brass pen, leather key ring, and a houseplant.',
+        href: '#',
+    },
+    {
+        name: 'Travel',
+        description: 'Daily commute essentials',
+        imageSrc: 'https://tailwindui.com/img/ecommerce-images/home-page-02-edition-03.jpg',
+        imageAlt: 'Collection of four insulated travel bottles on wooden shelf.',
+        href: '#',
+    },
+]
+
+export default function Hp() {
```

Comment:
Maybe we should rename this so it can be more intuitive what this file / component is doing